api/core/rag/datasource/vdb/pgvector/pgvector.py

```diff
@@ -57,6 +57,10 @@ def validate_config(cls, values: dict) -> dict:
 ) using heap;
 """
 
+SQL_CREATE_INDEX = """
+CREATE INDEX IF NOT EXISTS embedding_cosine_v1_idx ON {table_name} USING hnsw (embedding vector_cosine_ops);
+"""
```

Comment:
Prevent to use global variables for less memory consumpsion. Consider to use in-line variable inside the function instead.